### ISSUE TITLE
Upgrading to 0.3.7 fails

### ISSUE THREAD

**Issue by blairham** [loft-sh/loft#35]
When I tried to upgrade to the latest version of loft '0.3.7' I get the following error:

```bash
Error: UPGRADE FAILED: template: loft/templates/deployment.yaml:32:22: executing "loft/templates/deployment.yaml" at <.Values.livenessProbe.enabled>: nil pointer evaluating interface {}.enabled
```

The following command was used:
`helm upgrade -n loft loft loft --repo https://charts.devspace.sh --reuse-values --version 0.3.7`

**Comment by LukasGentele** [comment]
Yeah, so, the problem is that we added new default values in values.yaml for this chart and `--reuse-values` unfortunately does not do a merge of your old values with the standard default values. As a result, every new entry in the chart's default values.yaml is nil although the chart defines defaults for this value. We could add a lot of `{{if x}}` statements to the templates as a solution but maybe there is smarter way to solve this via Helm. I'll need to take a closer but wanted to give you a quick insight about the cause of this issue. Thanks for reporting this, @blairham!

**Comment by LukasGentele** [comment]
Here is [a discussion on this issue in helm v3](https://github.com/helm/helm/issues/8085) and a workaround that was already posted for v2 but should work for helm v3 as well: https://github.com/helm/helm/issues/3655#issuecomment-376478785

**Comment by blairham** [comment]
I was able to upgrade from 0.3.6 to 0.3.7 using the following command:
```bash
helm -n loft get values loft --all > values.yaml && helm upgrade -n loft loft loft --repo https://charts.devspace.sh --version 0.3.7 -f values.yaml
```

**Comment by FabianKramm** [comment]
@blairham should be fixed with v0.5.0
